### ISSUE TITLE
Django Admin - Tooltip improvements for wider compatibility 

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1413]
**Description:**
As a django admin user using Chrome (or other browsers with the same <title> support) I want keyboard navigable tooltips. Currently, the tooltips only work with mouse. 

Ticket update notes: 
Adjusted AC from original to reflect that we are indeed shifting away from tooltips in favor of span text due to tabindex changes not impacting the behavior we were seeing. (Edits by Miles on 1/4/2022)

**Acceptance Criteria:** 
- [ ] Span text containing tooltip text is visible across all breakpoints
- [ ] Span text is read by screenreaders via aria-describeby when associated heading is read (e.g. Available Permissions and Chosen Permissions)
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
- [x] Add tabIndex=0 to relevant link elements
- [x] implement tooltips as span
- [x] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
- Working theory is that giving tooltips a "tab-index=0" should correct for inconsistent title attribute support ([Chrome Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=829352&can=2&q=os%3Dchrome))
[HackMD notes](https://hackmd.io/3zR7vj7GQ7uidJmLR-eCYw?view#Follow-On-Stuff)
- If the above doesn't fix it then we could just spin out the tooltip text as span text on the page. 
- Noting that there are other accessible tooltip implementations we could turn to instead of span text or the tab index fix but these would be significantly more of a lift than either to implement.
- Investigate `help-tooltip.js` and `change_form.html` in Django-admin-508.

**Open Questions:**
- Does Robert concur/have other preferences for implementing a different accessible tooltip vs adding span text if the tab index fix doesn't work?

![image](https://user-images.githubusercontent.com/28515874/140803098-57b00f19-de2b-4945-8ffd-3b7968b1bfbb.png)
Example of keyboard focused tooltip behavior (outside of VO)

![image](https://user-images.githubusercontent.com/28515874/148086911-b5082c3f-efd1-4b03-8c7a-2b0b29599324.png)
Example of how span text solution should appear

**Comment by ADPennington** [comment]
11/17 Notes:
- Alt strategy (span text) okay? Check for consistency across browsers during testing for primary strategy. Else, span text alternative is good. More real estate but its clear. Could potentially using accordion.

**Comment by jorgegonzalez** [comment]
@riatzukiza and I cannot reproduce this on non-Windows environments.

**Comment by valcollignon** [comment]
AC and tasks need to be updated. @reitermb to add updates/image(s). 

Pausing on the work over the holiday sprint and will resume in January. Per @riatzukiza during sprint planning/backlog refinement 12.21.21.

**Comment by reitermb** [comment]
> AC and tasks need to be updated. @reitermb to add updates/image(s).
> 
> Pausing on the work over the holiday sprint and will resume in January. Per @riatzukiza during sprint planning/backlog refinement 12.21.21.

Updated!

**Comment by riatzukiza** [comment]
Thanks @reitermb, This'll be my first issue for the sprint.
